### PR TITLE
recreated the semesters again

### PR DESCRIPTION
**Pull request by MiniViolin**

### CODE REVIEW

File: orion/sites/admin/src/index.js [hunk 1]
```diff
@@ -32,7 +34,11 @@ const ProgramEdit = () => <ProgramEditPage/>;
 const ProgramEditMatch = ({match}) => <ProgramEditPage programId={match.params.programId}/>;
 const Location = () => <LocationPage/>;
 const LocationEdit = () => <LocationEditPage/>;
+const LocationEditMatch = ({match}) => <LocationEditPage locationId={match.params.locationId}/>;
```
Comment (ahsu1230) [inline orion/sites/admin/src/index.js]: looks like duplicates here. I think this line goes away

Comment (ahsu1230) [reply orion/sites/admin/src/index.js]: so keep 38 but remove 37

File: orion/sites/admin/src/index.js [hunk 2]
```diff
@@ -63,6 +69,9 @@ class App extends React.Component {
           <Route path="/locations/:locId/edit" component={LocationEditMatch}/>
           <Route path="/locations/add" component={LocationEdit}/>
           <Route path="/locations" component={Location}/>
+          <Route path="/semesters/:locationId/edit" component={SemesterEditMatch}/>
```
Comment (ahsu1230) [inline orion/sites/admin/src/index.js]: You have `:locationId` here. Probably `semesterId`?

Comment (MiniViolin) [reply orion/sites/admin/src/index.js]: Oops

**Pull request merged**